src/main/java/org/tron/core/db/Manager.java

```diff
@@ -145,6 +150,42 @@ public boolean processTrx(TransactionCapsule trxCap) {
     return true;
   }
 
+  private void voteWitessCount(Transaction trx) {
+    try {
+      int parameterSize = trx.getParameterList().size();
+      if (parameterSize <= 0) {
+        logger.info("transaction parameter list size is {}", parameterSize);
+        return;
+      }
+      Any parameter = trx.getParameterList().get(0);
+      if (parameter.is(VoteContract.class)) {
+        VoteContract voteContract = parameter.unpack(VoteContract.class);
+        int voteAdd = voteContract.getCount();
+        if (voteAdd > 0) {
+          voteContract.getVoteAddressList().forEach(voteAddress -> {
+            countVoteWitess(voteAddress, voteAdd);
+          });
+        }
+      }
+    } catch (InvalidProtocolBufferException e) {
+      e.printStackTrace();
+    }
+  }
+
+  private void countVoteWitess(ByteString voteAddress, int countAdd) {
```

Comment:
`countVoteWitess` -> `countVoteWitness`

Reply:
All right, this is really my mistake, thank you for your correction, and I will fix it immediately.
@KamilZielinski